addons/mail/static/src/backend_components/many2one_avatar_user/many2one_avatar_user.js

```diff
@@ -0,0 +1,96 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { Many2OneField } from '@web/views/fields/many2one/many2one_field';
+import { useCommand } from "@web/core/commands/command_hook";
+
+const { Component } = owl;
+
+export class Many2oneAvatarUser extends Many2OneField {
+
+    setup() {
+        super.setup();
+        const provide = async (env, options) => {
+            const records = await this._searchAssignTo(options.searchValue, 10);
+            return records.map((record) => ({
+                name: record[1],
+                action: () => {
+                    this.update([{ id: record[0], display_name: record[1] }]);
+                }
+            }));
+        };
+        useCommand(
+            this.env._t('Assign to ...'),
+            () => {
+                return {
+                    configByNamespace: {
+                        default: {
+                            emptyMessage: this.env._t("No users found"),
+                        },
+                    },
+                    placeholder: this.env._t("Select a user..."),
+                    providers: [{ provide }],
+                };
+            },
+            {
+                category: "smart_action",
+                hotkey: "alt+i",
+                global: true
+            }
+        );
+        useCommand(
+            this.env._t("Assign/unassign to me"),
+            () => {
+                if (this.env.services.user.userId === this.props.value[0]) {
+                    this.update([{}]);
+                } else {
+                    this.update([{ id: this.env.services.user.userId, display_name: this.env.services.user.name }]);
+                }
+            },
+            {
+                category: "smart_action",
+                hotkey: "alt+shift+i",
+                global: true,
+            }
+        );
+    }
```

Comment:
Are you sure this field is only used in FormView? These features were only available in FormView, and it looks like this field is also used in other views like Kanban.